### DISCUSSION TITLE
How can I disable dependabot for my forked repo?

### DISCUSSION THREAD

**Discussion by rohit2sharma95** [twbs/bootstrap#31385] [General] [answered]
Because you guys have already configured the dependabot on your repo, I am also receiving the pull requests from dependabot in my forked [repo](https://github.com/rohit2sharma95/bootstrap).

But I don't want these pull requests because the updates are being done already in the upstream repo. So merging those PRs does not make any sense, I can take them from upstream.

If I delete **.github/dependabot.yml** file it will make difference in my master branch and I can not delete this repo in my dependabot dashboard, there is no settings icon.
![Screenshot from 2020-07-31 17-46-05](https://user-images.githubusercontent.com/26922866/89036059-a448b580-d359-11ea-9e4b-364539fb3657.png)

I know, this question should not be asked here but maybe I can get any solution for it from your end.

**Comment by Johann-S** [comment]
You should go to dependabot website and disable it

**Reply by rohit2sharma95** [reply]
Are you talking about https://app.dependabot.com/?
I can not disable it from there, settings do not apply to the repositories that have been migrated to the GitHub-native version of Dependabot :)

**Comment by XhmikosR** [comment, marked as answer]
@rohit2sharma95 unfortunately, there no way though the UI. We are also having the same issue for our internal forks.

What you can do is change the default branch to a branch which doesn't have a dependabot.yml file.

I've been meaning to send GitHub support an email about this but never got the time to do it. Feel free to do it, though, because I'm pretty sure we are not the only ones with this issue.

**Comment by XhmikosR** [comment]
You should ask GitHub not us.
